backend/corpora/common/entities/project.py

```diff
@@ -0,0 +1,146 @@
+import typing
+
+from ..corpora_orm import (
+    Base,
+    DbProject,
+    DbProjectLink,
+    DbProjectDataset,
+    DbDatasetContributor,
+    DbContributor,
+    ProjectStatus,
+)
+from .entity import Entity
+from ..utils.exceptions import CorporaException
+
+
+class Project(Entity):
+    def __init__(self, **kwargs):
```

Comment:
There is a lot of redundant packings and unpackings going on. From `_parse` to `__init__` the project variables are packed twice and unpacked twice. It's not efficient. I suggest one of the following:
1.  Naming each argument expected in `__init__` to avoid having to unpack `**kwargs` Then use `**param` on line 125. 
1. Alternatively, pass in `param` from line 125 and unpack it in `__init__`. 
1. Another suggestion is to eliminate `_load` all together and that will reduce the amount of shuffling in half. The output from `_parse` can go straight into `__init__`.

My preference is for 3.

Reply:
Fair point on packing and unpacking. A combination of 1 and 3 seems reasonable! The way `Project` is currently being instantiated felt bloated to add to `_parse` which can get unwieldy. With one, it looks like that can be done in 1 line which I'm ok with.

Reply:
Done